README.md

```diff
@@ -8,23 +8,23 @@
 [![Doc](https://readthedocs.org/projects/dishka/badge/?version=latest&style=flat)](https://dishka.readthedocs.io)
 [![Telegram](https://img.shields.io/badge/💬-Telegram-blue)](https://t.me/reagento_ru)
 
-Cute DI framework with scopes and agreeable API.
+A Cute DI framework with scopes and agreeable API.
 
 📚 [Documentation](https://dishka.readthedocs.io)
 
 ### Purpose
 
-This library is targeting to provide only an IoC-container but make it really useful. If you are tired manually passing objects to create others objects which are only used to create more objects - we have a solution. Not all project require an IoC-container, but check what we have.
+This library's target is providing you with a single IoC-container and making it really useful. If you are tired of passing objects manually to create other objects which are only used to create more objects - we have a solution. Not all projects require an IoC-container, but check what we have.
```

Comment:
First sentence looks really different now. "Single" can be misunderstood: we support multiple containers, but we do not think about anything other than making good containers.